src/main/java/taxi/controllers/car/AddCarController.java

```diff
@@ -0,0 +1,37 @@
+package taxi.controllers.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.lib.Injector;
+import taxi.model.Car;
+import taxi.model.Manufacturer;
+import taxi.service.CarService;
+import taxi.service.ManufacturerService;
+
+public class AddCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("taxi");
```

Comment:
```suggestion
    private static final Injector INJECTOR = Injector.getInstance("taxi");
```
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html#:~:text=Class%20names%20should%20be%20nouns,such%20as%20URL%20or%20HTML